examples/GCH-Roy.ipynb

```diff
@@ -0,0 +1,562 @@
+       "            </div>\n",
+       "        "
+      ],
+      "text/plain": [
+       "Canvas(toolbar=Toolbar(toolitems=[('Home', 'Reset original view', 'home', 'home'), ('Back', 'Back to previous …"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.scatter(density,energy,c=dch_dist, marker='.')\n",
+    "ssel = sel[np.argsort(density[sel])]\n",
+    "plt.plot(density[ssel], energy[ssel], 'k--')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "ea1bb42f-7aca-4224-8a87-afada4a5b0c3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Mean hull energy for 'known' stable structures 1.816657381014075 kca/mol\n",
+      "Mean hull energy for 'other' structures 6.312730486304906 kca/mol\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"Mean hull energy for 'known' stable structures {dch_dist[iknown].mean()} kca/mol\")\n",
+    "print(f\"Mean hull energy for 'other' structures {dch_dist[iothers].mean()} kca/mol\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bc30f3e4-509b-40e6-b684-722d22cd44eb",
+   "metadata": {},
+   "source": [
+    "You can also visualize the hull with `chemiscope`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "1a9bfff8-31ed-49ad-81d0-d7530eb0b220",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "bd1119982d5648b1b557fd9090721538",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "ChemiscopeWidget(value='{\"meta\": {\"name\": \" \"}, \"settings\": {\"map\": {\"x\": {\"property\": \"density\"}, \"y\": {\"prop…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "try:\n",
+    "    import chemiscope\n",
```

Comment:
I would just let it run without the block. This is the last block of your example and we can add chemiscope as an example dependency.